src/OrchardCore.Modules/OrchardCore.Cors/Views/CorsSettings.Edit.cshtml

```diff
@@ -0,0 +1,216 @@
+                    <span class="hint">@T["Allows requests with any method. This will bypass any methods that are configured below."]</span>
+                </fieldset>
+
+                <options-list v-bind:options="policy.AllowedMethods" optionType="@T["Method"]" title="@T["Allowed methods"]" subTitle="" />
+            </div>
+        </div>
+
+        <fieldset class="form-group">
+            <button type="button" class="btn btn-primary" v-on:click="$emit('ok', policy, $event)">Ok</button>
+            <button type="button" class="btn btn-warning" v-on:click="$emit('cancel')">Cancel</button>
+        </fieldset>
+    </div>
+</script>
+
+<script type="text/x-template" id="options-list">
+    <div>
+        <label>{{ title }}</label>
+        <span class="hint" v-if="subTitle">{{ subTitle }}</span>
+        <nav class="admin-toolbar">
+            <ul class="navbar-nav mr-auto">
+                <li class="nav-item">
+                    <div class="row">
+                        <div class="col">
+                            <input v-model="newOption" type="text" class="form-control" />
+                        </div>
+                        <div class="col">
+                            <button type="button" class="btn btn-primary btn-sm" v-on:click="addOption(newOption)">@T["Add"] {{ optionType }}</button>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </nav>
+
+        <ul class="list-group" v-for="option in options">
+            <li class="list-group-item">
+                <div class="properties">
+                    <div class="related float-xs-right">
+                        <button type="button" class="btn btn-danger btn-sm" v-on:click="deleteOption(option)">@T["Delete"]</button>
+                    </div>
+                    {{ option }}
+                </div>
+            </li>
+        </ul>
+    </div>
+
+</script>
+
+<script at="Foot">
+    var optionsList = Vue.component('options-list',
+        {
+            props: ['options', 'optionType', 'title', 'subTitle'],
+            template: '#options-list',
+            data: function() {
+                return {
+                    'newOption' : ''
+                };
+            },
+            methods: {
+                addOption: function(value) {
+                    this.options.push(value);
+                },
+                deleteOption: function(value) {
+                    this.options.splice($.inArray(value[0], this.options), 1);
+                }
+            }
+        });
+
+    var policyDetails = Vue.component('policy-details',
+        {
+            components: { optionsList },
+            props: ['policy'],
+            template: '#policy-details'
+        });
+
+    var corsApp = new Vue({
+        el: '#corsAdmin',
+        components: { policyDetails, optionsList },
+        data: {
+            policies: @Html.Raw(JsonConvert.SerializeObject(Model.Policies)),
```

Comment:
My suggested change avoids the risk of XSS attack according to https://stackoverflow.com/a/4072787
```suggestion
            policies: @Html.Raw(Json.Encode(Model.Policies)),
```

Reply:
I used Serialize since I can't find Encode